HACKING.md

```diff
@@ -0,0 +1,27 @@
+⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
+
+# Hacking on the MCD
+
+1. Create a cluster (e.g. using [libvirt](https://github.com/openshift/installer/blob/master/Documentation/dev/libvirt-howto.md))
+2. Copy `/opt/tectonic/node-annotations.json` to `/etc/machine-config-daemon/` file on each node and pass it over this substitution:
+
+```
+sed -i s/node-configuration.v1.coreos.com/machineconfiguration.openshift.io/g /etc/machine-config-daemon/node-annotations.json
```

Comment:
In fact, you can drop the "Copy `/opt/tectonic/node-annotations.json` to..." line entirely and use:

```markdown
2. Run:

    ```sh
    sed s/node-configuration.v1.coreos.com/machineconfiguration.openshift.io/g /opt/tectonic/node-annotations.json >/etc/machine-config-daemon/node-annotations.json
    ```
```

Then the target file is never on-disk without the substitution.

The hanging indents in my suggestion will also make the command being executed part of this list entry.  With your current form, it leaks out:

```console
$ curl -s https://github.com/openshift/machine-config-operator/blob/2da5a0069cd9c72606f9a4ec0035cfeefa1b60d2/HACKING.md | grep -3 'sed -i s/node-'
<li>Create a cluster (e.g. using <a href="https://github.com/openshift/installer/blob/master/Documentation/dev/libvirt-howto.md">libvirt</a>)</li>
<li>Copy <code>/opt/tectonic/node-annotations.json</code> to <code>/etc/machine-config-daemon/</code> file on each node and pass it over this substitution:</li>
</ol>
<pre><code>sed -i s/node-configuration.v1.coreos.com/machineconfiguration.openshift.io/g /etc/machine-config-daemon/node-annotations.json
</code></pre>
<p>The install process will take care of this eventually.</p>
<ol start="3">
```